### ISSUE TITLE
Finish all variables

### ISSUE THREAD

**Issue by williamspiro** [williamspiro/HubL-Language-Extension#11]
https://designers.hubspot.com/docs/hubl/hubl-supported-variables

Related to https://github.com/williamspiro/HubL-Language-Extension/issues/10

See https://github.com/williamspiro/HubL-Language-Extension/tree/master/snippets/man_gen for what has been done

**Comment by TheWebTech** [comment]
I'll claim all of the email variables in the email variables block on the docs.

**Comment by TheWebTech** [comment]
Also claiming all of the Color and font settings

I accidentally did most of the work for this section too anyways.

**Comment by williamspiro** [comment]
@TheWebTech thank you so much!!

Can you create unique files in `/snippets/man_gen` for those which follow the same naming conventions? Want to make sure and keep them organized. Then, you add those new files to the package.json file here https://github.com/williamspiro/HubL-Language-Extension/blob/master/package.json#L19

**Comment by TheWebTech** [comment]
Yep will do, already started it that way. though i didn't know about the package.json needing to be updated. I'll do that

**Comment by brandenrodgers** [comment]
Just checking in, what's the status of this? Can we close now?

**Comment by TheWebTech** [comment]
Worked on it before joining HS. Not certain if anyone added the other vars.

If we take another stab at getting all the vars, we should consider finding a way to dynamically support these. That way no one needs to manually update/create all of them.
